test/factory/common_factory_tests.dart

```diff
@@ -10,7 +10,7 @@ import "package:react/react_client/js_interop_helpers.dart";
 
 import '../util.dart';
 
-void commonFactoryTests(Function factory) {
+void commonFactoryTests(/*Function*/dynamic factory) {
```

Comment:
Can we type this as `ReactComponentFactoryProxy` instead? Same goes for other usages in this file